scripts/runfabtests.sh

```diff
@@ -366,12 +379,26 @@ done
 # shift past options
 shift $((OPTIND-1))
 
-if [[ $# -ne 3 ]]; then
+if [[ $# -ge 4 ]]; then
 	usage
 fi
 
-PROV=$1
-SERVER=$2
-CLIENT=$3
+if [[ $# -ge 1 ]]; then
+	PROV=$1
+fi
+
+if [[ $# -ge 2 ]]; then
+	SERVER=$2
+	SSH_SERVER="${ssh} ${SERVER}"
+else
+	SSH_SERVER=eval
```

Comment:
nit: just set ssh_{client,server} default values (eval) at the top and skip both of these else conditions.